### DISCUSSION TITLE
Collections from materialized views

### DISCUSSION THREAD

**Discussion by joconnor-ecaa** [developmentseed/tipg#170] [Q&A] [answered]
Hi all, thank you for the excellent work.

I'd like to know if I can serve vector tiles from a materialized view in postgres. I have tipg running successfully against multiple tables in my DB, but it does not pick up any views. Is this something I can fix with configuration?

**Comment by vincentsarago** [comment, marked as answer]
Materialized view should be available if they are in the schema defined with `TIPG_DB_SCHEMA` env (default to `public`)

The `catalog` code is over https://github.com/developmentseed/tipg/blob/main/tipg/sql/dbcatalog.sql#L251-L260 if you want more hints. 

we do tests this with https://github.com/developmentseed/tipg/blob/main/tests/fixtures/my_data.sql#L18-L19

also make sure that the view as a geometry column declared
